crates/holochain/src/conductor/api/api_cell.rs

```diff
@@ -104,12 +104,12 @@ impl CellConductorApiT for CellConductorApi {
             .get_ribosome(self.cell_id.dna_hash())?)
     }
 
+    #[tracing::instrument(skip(self))]
```

Comment:
Well, because `self` is not `Debug`. If someone is debugging and needs extra info they can add `fields` to the instrumentation or extra tracing.

Are you proposing we include the maximum possible info in trace spans?